### PR TITLE
add UI checkbox for setting skip_invoicing_if_no_feature_charges

### PR DESCRIPTION
**Pull request by nickpell**
@proteusvacuum

### CODE REVIEW

File: corehq/apps/accounting/forms.py [hunk 1]
```diff
@@ -494,6 +497,7 @@ def __init__(self, subscription, account_id, web_user, *args, **kwargs):
             self.fields['do_not_email_invoice'].initial = subscription.do_not_email_invoice
             self.fields['do_not_email_reminder'].initial = subscription.do_not_email_reminder
             self.fields['auto_generate_credits'].initial = subscription.auto_generate_credits
+            self.fields['skip_invoicing_if_no_feature_charges'].initial = subscription.skip_invoicing_if_no_feature_charges
```
Comment (dimagimon) [inline corehq/apps/accounting/forms.py]: E501 line too long (123 > 115 characters)

**Comment by proteusvacuum** [comment]
:+1: on :white_check_mark:

**Comment by nickpell** [comment]
✅

**Comment by nickpell** [comment]
> 👍 on ✅

self-merging

**Pull request merged**